src/contour/ui.template/Terminal.qml.in

```diff
@@ -28,6 +28,31 @@ ContourTerminal
         focus : false
     }
 
+
+
+    Rectangle {
+        Timer {
+            id: sizeWidgetTimer
+            interval: 1000;
+            running: false;
+            onTriggered: sizeWidget.visible = false
+        }
+        id : sizeWidget
+        anchors.centerIn: parent
+        width:  80
+        height: 30
+        color: "white"
```

Comment:
we should probably give it a border to, just in case, because some users have a white/bright background, which makes this notification box hard to see then

Reply:
added black border of 1 pixel width